reference/3.0/Microsoft.PowerShell.Core/About/about_Command_Precedence.md

```diff
@@ -40,16 +39,75 @@ following rules.
   path to the script file.
 
   To run a script that is in the current directory, specify the full path, or
-  type a dot `.` to represent the current directory.
+  type a dot `.\` to represent the current directory.
 
   For example, to run the FindDocs.ps1 file in the current directory, type:
 
   ```
   .\FindDocs.ps1
   ```
 
-- If you do not specify a path, PowerShell uses the following precedence order
-  when it runs commands:
+### Using wildcards in execution
+
+You may use wildcards in command execution. Using wildcard characters is
+also known as *globbing*.
+
+PowerShell will execute a file that has a wildcard match, before a literal
```

Comment:
Can we remove the "will" to make it less passive?

Reply:
done